pkg/apis/sources/v1alpha1/apiserver_types_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "testing"
+
+func TestApiServerSource_GetGroupVersionKind(t *testing.T) {
+	ass := ApiServerSource{}
+	gvk := ass.GetGroupVersionKind()
+	if gvk.Kind != "ApiServerSource" {
+		t.Errorf("Should be ApiServerSource.")
```

Comment:
Consider dumping the diff, or just change this to:
if ApiServerSource{}.GetGroupVersionKind() != "ApiServerSource" {